src/subprocess_test.cc

```diff
@@ -261,3 +261,17 @@ TEST_F(SubprocessTest, ReadStdin) {
   ASSERT_EQ(1u, subprocs_.finished_.size());
 }
 #endif  // _WIN32
+
+#ifndef _WIN32
+TEST_F(SubprocessTest, Background) {
+  time_t before = time(NULL);
+  Subprocess* subproc = subprocs_.Add("echo -n 123; sleep 10s&");
```

Comment:
I'm probably just being paranoid. The project I work on the most has had a lot of timing-related failures in the past, but they've tended to be more heavyweight than this.

I've reduced it to 2s. I'd make it 1s, but then the second might reasonably tick over between calls to time(). I could go and use gettimeofday for better resolution, but this is only matters for the theoretical failure case, so doesn't seem worthwhile complicating the test.